### ISSUE TITLE
Remove loaders

### ISSUE THREAD

**Issue by kousu** [ecometrica/django-multisite#41]
Our instructions [currently say](https://github.com/ecometrica/django-multisite/blob/6f3302af2b64baeb64fe06d61280c559e39ca3df/README.rst) to use `TEMPLATES['OPTIONS']['loaders']`, but this conflicts with [APP_DIRS](https://docs.djangoproject.com/en/1.10/ref/settings/#app-dirs) in Django 1.10:
```
?: (templates.E001) You have 'APP_DIRS': True in your TEMPLATES but also specify 'loaders' in OPTIONS. Either remove APP_DIRS or remove the 'loaders' option.
```

We need to update our instructions for Django 1.10 before #37 is really finished. This might require writing something like new backend templates to support `APP_DIRS`.

(we should probably also make the comment about `TEMPLATE_LOADERS` on pre-1.8 clearer)

**Comment by kousu** [comment]
Also you must have
```
INSTALLED_APPS = [
   ...
    'django.contrib.sites',
   ...
]
```
but this isn't documented.

**Comment by rebkwok** [comment]
Fixed in PR #45
